index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="container">
+      <p id="logo"><a href="index.html">AlierPM</a></p>
+      <nav>
+        <a href="#"><img src="../images/Union.png" alt="Mobile navigation menu"
+        /></a>
+      </nav>
```

Comment:
- [OPTIONAL] I suggest decreasing the width of the menu icon, to be similar to the Figma design. You can do this by adding the `width` attribute. 
Example:
```
<img src="../images/Union.png" alt="Mobile navigation menu" width="20px">
```

Reply:
>Thank you for your constructive review;  I will implement that suggestion. All the best in your journey too.